lib/authorized_persona/authorization.rb

```diff
@@ -14,14 +14,14 @@ module Authorization
     class_methods do
       # Configure authorization for an authorized persona class
       def authorize_persona(class_name:, current_user_method: nil) # rubocop:disable Metrics/AbcSize, Metrics/PerceivedComplexity, Metrics/
-        raise AuthorizedPersona::Error, "you can only configure authorization once" if authorization_persona_class_name.present?
-        raise AuthorizedPersona::Error, "class_name must be a string" unless class_name.is_a?(String)
-        raise AuthorizedPersona::Error, "current_user_method must be a symbol" if current_user_method && !current_user_method.is_a?(Symbol)
+        raise Error, "you can only configure authorization once" if authorization_persona_class_name.present?
+        raise Error, "class_name must be a string" unless class_name.is_a?(String)
+        raise Error, "current_user_method must be a symbol" if current_user_method && !current_user_method.is_a?(Symbol)
```

Comment:
__suggestion: (nitpick)__ The `nil` value also responds to `is_a?` so you probably don't need the explicit `nil` check. If this is changed to `unless` it can be written a bit more simply as:

```ruby
raise Error, "current_user_method must be a symbol" unless current_user_method.is_a?(Symbol)
```